BookingsApi/BookingsApi.DAL/Migrations/20220130152724_AddCaseHearingTypesForTenantFees.cs

```diff
@@ -0,0 +1,86 @@
+﻿using BookingsApi.DAL.Helper;
+using BookingsApi.Domain.Enumerations;
+using BookingsApi.Domain.RefData;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace BookingsApi.DAL.Migrations
+{
+    public partial class AddCaseHearingTypesForTenantFees : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
+                nameof(CaseType),
+                new[] { "Id", "Name" },
+                new object[,]
+                {
+                    {43, "Tenant Fees"}
+                });
+
+            migrationBuilder.InsertData(
+                nameof(HearingType),
+                new[] { "Id", "Name", "CaseTypeId" },
+                new object[,]
+                {
+                    {219, "Case Management", 43},
+                    {220, "Mediation", 43},
+                    {221, "Priliminary Hearing", 43},
```

Comment:
Might be a typo here - Preliminary instead of Priliminary?

Reply:
changed